packages/parser-css/src/parser.ts

```diff
@@ -0,0 +1,68 @@
+import * as postcss from 'postcss';
+
+import * as logger from 'hint/dist/src/lib/utils/logging';
+import { IAsyncHTMLElement, ElementFound, FetchEnd, Parser } from 'hint/dist/src/lib/types';
+import { StyleParse } from './types';
+import { Engine } from 'hint/dist/src/lib/engine';
+
+const styleContentRegex: RegExp = /^<style[^>]*>([\s\S]*)<\/style>$/;
```

Comment:
Something such as:

```
<style>

</style     >
         ⬆️ spaces
```

should also match, because: [`4. After the tag name, there may be one or more ASCII whitespace.`](https://html.spec.whatwg.org/#end-tags)